app/models/TaxYearSelection.scala

```diff
@@ -84,13 +89,69 @@ object TaxYearSelection extends Enumerable.Implicits {
     }
   }
 
+  def filterSelectedTaxYear(taxYearSelection: Seq[TaxYearSelection], claimYear: String): Seq[TaxYearSelection] = {
```

Comment:
can claimYear be passed in as a string as this is pulled from the params can be Int

Reply:
Resolve this for the current ticket and raise a tech debt ticket for the other instances of year